manual_jb/content/analyses/equity_metric.md

```diff
@@ -0,0 +1,81 @@
+# Equity Metric Analysis
```

Comment:
To be consistent with other analyses, can you remove the "Analysis" from the title?

Reply:
hmmm....I now see others do the same. I defer to you, what's your preference? It seems like redundancy to me, but since others do this as well, if it sounds better, we can leave it as is.

Reply:
Yes I like it concise. Removed the analysis :-)